app/helpers/most_focus_helper.rb

```diff
@@ -0,0 +1,47 @@
+def most_focus(data_type, profile_id)
+  data = []
+  user_id = profile_id || current_user.id
```

Comment:
this seems a little weird to me, why is `user_id` either a profile id or a user id?